test/test-PreviewProvider.js

```diff
@@ -68,7 +68,7 @@ exports.test_only_request_links_once = function*(assert) {
   });
 };
 
-exports.test_filter_urls = function*(assert) {
+exports.test_filter_urls = function(assert) {
```

Comment:
Unrelated, but now that my eyes are seeing it...
In the file above this, we use something like:
```js
exports["test blocklist basic format"] = function (assert) {
```

But in this file we use:
```js
exports.test_filter_urls = function (assert) {
```